packages/geo/package.json

```
@@ -20,16 +19,26 @@
 		"test": "yarn run lint && jest -w 1 --coverage",
 		"test:size": "size-limit",
 		"build-with-test": "yarn test && yarn run build",
-		"build:cjs": "node ./build es5 && webpack && webpack --config ./webpack.config.dev.js",
-		"build:esm": "node ./build es6",
-		"build:cjs:watch": "node ./build es5 --watch",
-		"build:esm:watch": "rimraf lib-esm && node ./build es6 --watch",
+		"build:cjs": "rimraf lib && tsc -m commonjs --outDir lib && webpack && webpack --config ./webpack.config.dev.js",
+		"build:esm": "rimraf lib-esm && tsc -m esnext --outDir lib-esm",
+		"build:cjs:watch": "rimraf lib && tsc -m commonjs --outDir lib --watch",
+		"build:esm:watch": "rimraf lib-esm && tsc -m esnext --outDir lib-esm --watch",
 		"build": "yarn clean && yarn build:esm && yarn run build:cjs",
 		"clean": "npm run clean:size && rimraf lib-esm lib dist",
 		"clean:size": "rimraf dual-publish-tmp tmp*",
 		"format": "echo \"Not implemented\"",
 		"lint": "tslint '{__tests__,src}/**/*.ts' && npm run ts-coverage",
-		"ts-coverage": "typescript-coverage-report -p ./tsconfig.build.json -t 86.56"
+		"ts-coverage": "typescript-coverage-report -p ./tsconfig.json -t 84.00"
+	},
+	"typesVersions": {
+		">=3.8": {
```

We recently made a change to set the floor at 4.0 (will be updated in other categories)

```suggestion
		">=4.0": {
```